app/user.go

```diff
@@ -1362,8 +1362,8 @@ func (a *App) GetPasswordRecoveryToken(token string) (*model.Token, *model.AppEr
 }
 
 func (a *App) DeleteToken(token *model.Token) *model.AppError {
-	if result := <-a.Srv.Store.Token().Delete(token.Token); result.Err != nil {
```

Comment:
Here you can just `return a.Srv.Store.Token().Delete(token.Token)`

Reply:
Thanks for the suggestion. I have made the changes.